plugins/inputs/logfile/tailersrc.go

```diff
@@ -227,7 +243,13 @@ func (ts *tailerSrc) runTail() {
 					offset: *fo,
 					src:    ts,
 				}
-				ts.outputFn(e)
+				if ts.filterFn != nil {
```

Comment:
(I wanted to leave this comment on line 230 above, but GitHub won't let me comment on unchanged lines...)

This is tricky, but what should we do about lines that are truncated because they go over the max event size?  I suppose we have no choice but to apply the regex to the truncated line?  If that's what we decide, then just leave a comment so future readers at least know we thought about it.

Reply:
Ahh I didn't even notice that this didn't account for truncation - assumed the truncation happened after. I'm of the opinion that trying to do the filtering on the log line before truncation is just going to end up biting us down the road more than only processing the filters on the truncated log line.

Will also try to cook up a unit test that illustrates behavior for truncated log lines